### ISSUE TITLE
Question about hw1 Q1.2 BC Agent Analysis

### ISSUE THREAD

**Issue by Kin-Zhang** [mdeib/berkeley-deep-RL-pytorch-solutions#4]
![image](https://user-images.githubusercontent.com/35365764/118429270-e46bff80-b703-11eb-904e-8ef541bb9ef2.png)

As the picture shown here, I'm wondering how tensorboard can draw different event data with user horizontal axis...
horizontal axis by different train step inside the tensorboard
since you need to run different loops and it's different log files...

**Comment by Kin-Zhang** [comment]
And I review the hw1 code again, the train data set parameter didn't use in BC agent since we just have 1 iteration as code say here:
```python
        logdir_prefix = 'q1_'
        assert args.n_iter==1, ('Vanilla behavior cloning collects expert data just once (n_iter=1)')
```
And if we set the `n_iter=1` , it will only collecting data for eval but not training

In rl_train.py file:
```python
        # DONE TODO decide whether to load training data or use the current policy to collect more data
        # HINT: depending on if it's the first iteration or not, decide whether to either
                # (1) load the data. In this case you can directly return as follows
                # ``` return loaded_paths, 0, None ```

                # (2) collect `self.params['batch_size']` transitions
        if itr==0:
            with open(load_initial_expertdata, 'rb') as f:
                loaded_paths = pickle.loads(f.read())
            return loaded_paths, 0, None
```
it directly return, and since there are no next itr, it shouldn't work out in BC agent in Q 1.2

**Comment by mdeib** [comment]
Sorry for the late reply - When you log values into tensorboard you can set the x-values to whatever you would like - in this case they are just set to the training step that that particular evaluation was done at. The chunk of code in your second comment is just to load in expert data and is unrelated to the tensorboard logging, unless I am misunderstanding your question?
